Podfile

```diff
@@ -1,10 +1,9 @@
-source 'https://github.com/twilio/cocoapod-specs'
 source 'https://github.com/CocoaPods/Specs.git'
 
 workspace 'ObjCVoiceQuickstart'
 
 abstract_target 'TwilioVoice' do
-  pod 'TwilioVoice', '2.0.0'
+  pod 'TwilioVoice', '~> 2.0'
```

Comment:
You may want to pin to patch versions only `~> 2.0.0`. I would expect that a minor version bump would require at least some change the QS to demonstrate new features.